sharding/nodesCoordinator/nodesCoordinatorRegistryFactory.go

```diff
@@ -34,10 +34,11 @@ func NewNodesCoordinatorRegistryFactory(
 func (ncf *nodesCoordinatorRegistryFactory) CreateNodesCoordinatorRegistry(buff []byte) (NodesCoordinatorRegistryHandler, error) {
 	registry, err := ncf.createRegistryWithAuction(buff)
 	if err == nil {
-		log.Debug("nodesCoordinatorRegistryFactory.CreateNodesCoordinatorRegistry created registry with auction")
+		log.Debug("nodesCoordinatorRegistryFactory.CreateNodesCoordinatorRegistry created registry with auction",
+			"epoch", registry.CurrentEpoch)
 		return registry, nil
 	}
-	log.Debug("nodesCoordinatorRegistryFactory.CreateNodesCoordinatorRegistry created old registry")
+	log.Debug("nodesCoordinatorRegistryFactory.CreateNodesCoordinatorRegistry creating old registry")
```

Comment:
Cannot, since I don't have the epoch yet, only the buffer.
This is the reason why I've added the 2 logs:
1. nodesCoordinatorRegistryFactory.CreateNodesCoordinatorRegistry **creating** old registry
2. nodesCoordinatorRegistryFactory.CreateNodesCoordinatorRegistry **created** old registry, "epoch"